superset/reports/schemas.py

```diff
@@ -150,6 +156,12 @@ class ReportSchedulePostSchema(Schema):
         description=sql_description, example="SELECT value FROM time_series_table"
     )
     chart = fields.Integer(required=False, allow_none=True)
+    creation_method = EnumField(
+        ReportCreationMethodType,
+        by_value=True,
+        required=False,
```

Comment:
This will default to `ALERTS_REPORTS` if the field is missing, right?

Reply:
yes, so should I make it required?